teams/roles-and-responsibilities.md

```diff
@@ -1 +1,65 @@
-# Teams: Roles and Responsibilities
+# Ember Core Team Member Roles, Responsibilities, and Expectations
+
+## Participation
+
+As a bare minimum, core team members are expected to:
+
+- Follow the community guidelines- https://emberjs.com/guidelines 
+- Interact thoughtfully:
+    - with the community in an exemplary way - for better or for worse, the “core team” is often perceived as a unit, not as individuals
+    - with other members of the core teams in a respectful way (do we need to say this?)
+- Lead the official projects through some combination of
+    - Triaging issues
+    - Reviewing PRs and mentoring community contributors
+    - Answering community questions (Discord, [Discuss](https://discuss.emberjs.com/), Stack Overflow, etc)
+    - Supporting community documentation efforts
+    - Writing blog posts for emberjs.com
+    - Championing RFCs
+    - Supporting the release process
+- Attend weekly team meetings
+- Be reliable
+    - If you say you can do something, do it
+    - If you can’t do it (life happens- and we are all volunteers) let the team know
+
+
+## Annual Commitments
+
+The expectations for participation is that every core team member is expected to participate, in the way(s) that they define for themselves. 
```

Comment:
I absolutely have no clue what this is actually saying. Its much too vague, also doesn't this document _just above_ prescribe exactly how they are supposed to be participating?